nginx/ngx_js_fetch.c

```diff
@@ -287,6 +288,16 @@ ngx_js_http_resolve_done(ngx_js_http_t *http)
 }
 
 
+njs_inline void
+ngx_js_http_close_peer(ngx_js_http_t *http)
```

Comment:
Let's make it an ordinary static function and move somewhere down. 
It is up to compiler to inline it.